File: 프로그래머스/unrated/138477. 명예의 전당 （1）/명예의 전당 （1）.py
def solution(k, score):
    answer = []
    ary = []
    
    for i in range(0,len(score)):
        
        if i >= k :
            if ary[k-1] < score[i]:
                ary.append(score[i])   
                ary.sort(reverse = True)
                answer.append(ary[k-1])
            else:
                answer.append(ary[k-1])    
        else:
            ary.append(score[i])   
            ary.sort(reverse = True)
            answer.append(ary[i])
    return answer

File: 프로그래머스/1/138477. 명예의 전당 （1）/명예의 전당 （1）.js
function solution(k, score) {
    let result = []
    let store = []
    for(let i=0; i<score.length; i++) {
        store.push(score[i])
        store.sort((a,b) => b-a)
        
        if(i < k) {
            result.push(store[i])
        } else {
            result.push(store[k-1])
        }
    }
    return result
}